Loading rhel8/default-icl
  Loading requirement: dot rhel8/slurm singularity/current rhel8/global
    cuda/11.4 intel/compilers/2020.2 intel/mkl/2020.2 intel/impi/2020.2/intel
    intel/libs/idb/2020.2 intel/libs/tbb/2020.2 intel/libs/ipp/2020.2
    intel/libs/daal/2020.2 intel/bundles/complib/2020.2
WARNING: Logging before flag parsing goes to stderr.
W0518 15:00:02.855263 23258215949184 lazy_loader.py:50] 
The TensorFlow contrib module will not be included in TensorFlow 2.0.
For more information, please see:
  * https://github.com/tensorflow/community/blob/master/rfcs/20180907-contrib-sunset.md
  * https://github.com/tensorflow/addons
  * https://github.com/tensorflow/io (for I/O related ops)
If you depend on functionality not listed there, please file an issue.

Warning: Flow failed to import. Set the environment variable D4RL_SUPPRESS_IMPORT_ERROR=1 to suppress this message.
No module named 'flow'
Warning: CARLA failed to import. Set the environment variable D4RL_SUPPRESS_IMPORT_ERROR=1 to suppress this message.
No module named 'carla'
2022-05-18 15:00:06,977	INFO node.py:498 -- Process STDOUT and STDERR is being redirected to /tmp/ray/session_2022-05-18_15-00-06_977462_944383/logs.
2022-05-18 15:00:07,103	INFO services.py:409 -- Waiting for redis server at 127.0.0.1:38273 to respond...
2022-05-18 15:00:08,104	INFO services.py:414 -- Failed to connect to the redis server, retrying.
2022-05-18 15:00:08,105	INFO services.py:409 -- Waiting for redis server at 127.0.0.1:38273 to respond...
2022-05-18 15:00:08,256	INFO services.py:409 -- Waiting for redis server at 127.0.0.1:29850 to respond...
2022-05-18 15:00:08,257	INFO services.py:806 -- Starting Redis shard with 10.0 GB max memory.
2022-05-18 15:00:08,289	INFO node.py:512 -- Process STDOUT and STDERR is being redirected to /tmp/ray/session_2022-05-18_15-00-06_977462_944383/logs.
2022-05-18 15:00:08,289	WARNING services.py:1294 -- Warning: Capping object memory store to 20.0GB. To increase this further, specify `object_store_memory` when calling ray.init() or ray start.
2022-05-18 15:00:08,290	INFO services.py:1442 -- Starting the Plasma object store with 20.0 GB memory using /dev/shm.
2022-05-18 15:00:09,126	INFO tune.py:61 -- Tip: to resume incomplete experiments, pass resume='prompt' or resume=True to run()
2022-05-18 15:00:09,126	INFO tune.py:232 -- Starting a new experiment.
W0518 15:00:09.155928 23258215949184 deprecation_wrapper.py:119] From /rds/user/ajc348/hpc-work/mopo/.env/lib/python3.6/site-packages/ray/tune/logger.py:131: The name tf.VERSION is deprecated. Please use tf.version.VERSION instead.

W0518 15:00:09.156167 23258215949184 deprecation_wrapper.py:119] From /rds/user/ajc348/hpc-work/mopo/.env/lib/python3.6/site-packages/ray/tune/logger.py:136: The name tf.summary.FileWriter is deprecated. Please use tf.compat.v1.summary.FileWriter instead.

W0518 15:13:48.693564 23258215949184 deprecation_wrapper.py:119] From /rds/user/ajc348/hpc-work/mopo/.env/lib/python3.6/site-packages/ray/tune/logger.py:114: The name tf.Summary is deprecated. Please use tf.compat.v1.Summary instead.

2022-05-18 15:13:48,777	WARNING util.py:64 -- The `process_trial` operation took 0.1538541316986084 seconds to complete, which may be a performance bottleneck.
2022-05-18 15:19:56,717	WARNING util.py:64 -- The `save_to_disk` operation took 1.8848521709442139 seconds to complete, which may be a performance bottleneck.
2022-05-18 15:19:56,720	WARNING util.py:64 -- The `process_trial` operation took 1.8963453769683838 seconds to complete, which may be a performance bottleneck.
2022-05-18 15:21:18,354	WARNING util.py:64 -- The `process_trial` operation took 0.10952019691467285 seconds to complete, which may be a performance bottleneck.
2022-05-18 15:26:52,070	WARNING util.py:64 -- The `save_to_disk` operation took 0.40686488151550293 seconds to complete, which may be a performance bottleneck.
2022-05-18 15:26:52,074	WARNING util.py:64 -- The `process_trial` operation took 0.41601109504699707 seconds to complete, which may be a performance bottleneck.
2022-05-18 15:33:50,947	WARNING util.py:64 -- The `save_to_disk` operation took 0.422821044921875 seconds to complete, which may be a performance bottleneck.
2022-05-18 15:33:50,948	WARNING util.py:64 -- The `process_trial` operation took 0.4302685260772705 seconds to complete, which may be a performance bottleneck.
2022-05-18 15:40:43,525	WARNING util.py:64 -- The `save_to_disk` operation took 0.37918663024902344 seconds to complete, which may be a performance bottleneck.
2022-05-18 15:40:43,527	WARNING util.py:64 -- The `process_trial` operation took 0.3863027095794678 seconds to complete, which may be a performance bottleneck.
2022-05-18 15:47:36,361	WARNING util.py:64 -- The `save_to_disk` operation took 0.4344449043273926 seconds to complete, which may be a performance bottleneck.
2022-05-18 15:47:36,363	WARNING util.py:64 -- The `process_trial` operation took 0.44138574600219727 seconds to complete, which may be a performance bottleneck.
2022-05-18 15:54:30,949	WARNING util.py:64 -- The `save_to_disk` operation took 0.4700033664703369 seconds to complete, which may be a performance bottleneck.
2022-05-18 15:54:30,950	WARNING util.py:64 -- The `process_trial` operation took 0.4769721031188965 seconds to complete, which may be a performance bottleneck.
2022-05-18 16:01:25,958	WARNING util.py:64 -- The `save_to_disk` operation took 0.3955867290496826 seconds to complete, which may be a performance bottleneck.
2022-05-18 16:01:25,959	WARNING util.py:64 -- The `process_trial` operation took 0.4024336338043213 seconds to complete, which may be a performance bottleneck.
2022-05-18 16:08:21,437	WARNING util.py:64 -- The `save_to_disk` operation took 0.37832021713256836 seconds to complete, which may be a performance bottleneck.
2022-05-18 16:08:21,440	WARNING util.py:64 -- The `process_trial` operation took 0.3869764804840088 seconds to complete, which may be a performance bottleneck.
2022-05-18 16:15:16,373	WARNING util.py:64 -- The `save_to_disk` operation took 0.3819754123687744 seconds to complete, which may be a performance bottleneck.
2022-05-18 16:15:16,375	WARNING util.py:64 -- The `process_trial` operation took 0.3889591693878174 seconds to complete, which may be a performance bottleneck.
2022-05-18 16:22:12,337	WARNING util.py:64 -- The `save_to_disk` operation took 0.4311065673828125 seconds to complete, which may be a performance bottleneck.
2022-05-18 16:22:12,339	WARNING util.py:64 -- The `process_trial` operation took 0.43822765350341797 seconds to complete, which may be a performance bottleneck.
slurmstepd: error: *** JOB 61332800 ON cpu-q-254 CANCELLED AT 2022-05-18T16:23:42 ***
